app/helpers/compute_resources_vms_helper.rb

```diff
@@ -142,8 +142,12 @@ def show_vm_name?
     controller_name != 'hosts' && controller_name != 'compute_attributes'
   end
 
-  def new_host?(host)
-    host.try(:new_record?)
+  def new_host?
+    if @host.nil?
```

Comment:
It's rendered in the compute_resource_selected controller action, which has a `@host`, but I don't believe it's required.  Checking the state of the f.object (Fog server) should be sufficient.